<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartGame</class>
 <widget class="QMainWindow" name="StartGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>690</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="illegalLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>690</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CBButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>560</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>C -&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CAButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>560</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>C -&gt; A</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTimelabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>660</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current game</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentMinLcdNumber">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>660</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>630</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ABButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>A -&gt; B</string>
    </property>
   </widget>
   <widget class="QLabel" name="winLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentSecLcdNumber">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>660</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ACButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>A -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BAButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>B -&gt; A</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="bestMinLcdNumber">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>690</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="gameLogLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>610</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Game log:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestTimeLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>690</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Best time</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gameLogTextBrowser">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BCButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>B -&gt; C</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="bestSecLcdNumber">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>690</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="steptextBrowser">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>630</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="stepnumber">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>629</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step Count:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>77</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>How to Play</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>closeButton</zorder>
   <zorder>illegalLabel</zorder>
   <zorder>newGameButton</zorder>
   <zorder>CBButton</zorder>
   <zorder>CAButton</zorder>
   <zorder>currentTimelabel</zorder>
   <zorder>currentMinLcdNumber</zorder>
   <zorder>timerLabel</zorder>
   <zorder>ABButton</zorder>
   <zorder>winLabel</zorder>
   <zorder>currentSecLcdNumber</zorder>
   <zorder>ACButton</zorder>
   <zorder>BAButton</zorder>
   <zorder>bestMinLcdNumber</zorder>
   <zorder>gameLogLabel</zorder>
   <zorder>bestTimeLabel</zorder>
   <zorder>gameLogTextBrowser</zorder>
   <zorder>BCButton</zorder>
   <zorder>bestSecLcdNumber</zorder>
   <zorder>steptextBrowser</zorder>
   <zorder>stepnumber</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>StartGame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>739</y>
    </hint>
    <hint type="destinationlabel">
     <x>891</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
